packages/editor/src/components/collab-sidebar/comments.js

```diff
@@ -101,8 +102,20 @@ function Thread( {
 	setFocusThread,
 	setShowCommentBoard,
 } ) {
-	const { flashBlock } = useDispatch( blockEditorStore );
+	const { toggleBlockHighlight } = useDispatch( blockEditorStore );
 	const relatedBlockElement = useBlockElement( thread.blockClientId );
+	const debouncedToggleBlockHighlight = useDebounce(
+		toggleBlockHighlight,
+		50
+	);
+
+	const onMouseEnter = useCallback( () => {
+		debouncedToggleBlockHighlight( thread.blockClientId, true );
+	}, [ thread.blockClientId, debouncedToggleBlockHighlight ] );
+
+	const onMouseLeave = useCallback( () => {
+		debouncedToggleBlockHighlight( thread.blockClientId, false );
+	}, [ thread.blockClientId, debouncedToggleBlockHighlight ] );
```

Comment:
I think we can drop the callback memoization, unless they are used in useEffect.

Reply:
Removed in 7ecacf5ee41dd2a05ce9231554c01697db03f26a